pkg/network/protocols/http/monitor.go

```diff
@@ -59,6 +62,10 @@ func NewMonitor(c *config.Config, offsets []manager.ConstantEditor, sockFD *ebpf
 		return nil, fmt.Errorf("http monitoring is disabled")
 	}
 
+	if !c.EnableHTTP2Monitoring {
```

Comment:
Both http2 and http have to be enabled for the monitor to exist?

Reply:
Hasan is right, let support both cases, and let's make sure we are not hurting the functionality